src/quick-lint-js/error.h

```diff
@@ -1139,6 +1139,12 @@
       { source_code_span continue_statement; },                                \
       .error(QLJS_TRANSLATABLE("continue can only be used inside of a loop"),  \
              continue_statement))                                              \
+                                                                               \
+  QLJS_ERROR_TYPE(                                                             \
+      error_redundant_semicolon_after_else, "E202",                            \
```

Comment:
Is there a convention for error/warning order and naming?

Reply:
> Is there a convention for error/warning order

Yes. Sort lexicographically by the struct name (`error_redundant_semicolon_after_else`)

> Is there a convention for [...] naming?

Aside from the `error_` prefix and snake case, no.